components/Buttons/examples/FloatingButtonExampleSwiftViewController.swift

```diff
@@ -15,13 +15,17 @@
 import UIKit
 
 import MaterialComponents.MaterialButtons
+import MaterialComponentsBeta.MaterialButtons_Theming
+import MaterialComponentsBeta.MaterialContainerScheme
 
 class FloatingButtonExampleSwiftViewController: UIViewController {
 
   let miniFloatingButton = MDCFloatingButton(frame: .zero, shape: .mini)
   let defaultFloatingButton = MDCFloatingButton()
   let largeIconFloatingButton = MDCFloatingButton()
 
+  var containerScheme = MDCContainerScheme()
```

Comment:
Please revert this file - looks like theming isn't used/needed.

Reply:
I now theme the buttons to match the other examples.